Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org/'
 
-gem 'synapse', github: 'democracyworks/synapse', branch: 'v0.12.1-democracyworks'
+gem 'synapse', '~> 0.13.8'
```

Comment:
It was a change to how it filtered matching docker images based on exposed ports or not. We PR'd it and they merged it long ago, which is when we should have unforked. I don't know what all has changed since then, but looking over the releases I didn't see any mention of breaking changes and it still seems to work with this version in local testing.